<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QMainWindow" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>251</width>
      <height>69</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: left;
color: rgb(33, 152, 175);
</string>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Docs/ico.png</normaloff>Docs/ico.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_modify">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>251</width>
      <height>69</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: left;
color: rgb(33, 152, 175);</string>
    </property>
    <property name="text">
     <string>MODIFY</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Docs/ico.png</normaloff>Docs/ico.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_search">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>251</width>
      <height>69</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: left;
color: rgb(33, 152, 175);</string>
    </property>
    <property name="text">
     <string>SEARCH</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Docs/ico.png</normaloff>Docs/ico.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_display">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>300</y>
      <width>251</width>
      <height>69</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: left;
color: rgb(33, 152, 175);</string>
    </property>
    <property name="text">
     <string>DISPLAY</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Docs/ico.png</normaloff>Docs/ico.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_about">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>370</y>
      <width>251</width>
      <height>69</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: left;
color: rgb(33, 152, 175);</string>
    </property>
    <property name="text">
     <string>ABOUT</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Docs/ico.png</normaloff>Docs/ico.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>440</y>
      <width>251</width>
      <height>69</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: left;
color: rgb(33, 152, 175);</string>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Docs/ico.png</normaloff>Docs/ico.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_date">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(1, 170, 216);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Date Label&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_commands">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>251</width>
      <height>69</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: left;
color: rgb(33, 152, 175);</string>
    </property>
    <property name="text">
     <string>COMMANDS</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Docs/ico.png</normaloff>Docs/ico.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>691</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(&quot;C:/Users/Dustin/Documents/GitHub/VILAN/VDATABASE/Docs/background.jpg&quot;);</string>
    </property>
   </widget>
   <zorder>widget</zorder>
   <zorder>pushButton_add</zorder>
   <zorder>pushButton_modify</zorder>
   <zorder>pushButton_search</zorder>
   <zorder>pushButton_display</zorder>
   <zorder>pushButton_about</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>label_date</zorder>
   <zorder>pushButton_commands</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
